.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - python-version: "3.6"
+          env:
+            TOXENV: py
+        - python-version: "3.7"
+          env:
+            TOXENV: py
+        - python-version: "3.8"
+          env:
+            TOXENV: py
+        - python-version: "3.9"
+          env:
+            TOXENV: py
+        - python-version: "3.10"
+          env:
+            TOXENV: py
+        - python-version: "3.7"
+          env:
+            TOXENV: mypy
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install system libraries
+      if: contains(matrix.python-version, 'pypy3.9')
+      run: |
+        sudo apt-get update
+        sudo apt-get install libxml2-dev libxslt-dev
```

Comment:
This step seems not needed here.

Reply:
Good catch! Fixed in 6ec36ee02f3eee6700dd79a763f5b4bf5d41e94c